content/docs/guides/components/modeldb.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "ModelDB"
+description ="ModelDB - A system to manage machine learning models"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+
+## Using ModelDB
+
+After ModelDB is deployed using kfctl.sh and the modeldb-db, modeldb-frontend and modeldb-backend pods are running - 
+
+1)  Port forward the modeldb-backend pod to port 6543 - 
+
+```
+kubectl get pods -n kubeflow # Find your modeldb-backend pod
+kubectl port-forward [modeldb-backend pod] 6543:6543 -n kubeflow
+``` 
+2)  Create a ModelDB syncer
+
+ModelDBSyncer is the object that logs models and operations to the ModelDB backend. You can initialize the Syncer with your specified configurations as shown below. Explore the ModelDBSyncer here for more details on the Syncer object and the different ways to initialize it.
+
+You can initialize the syncer either from a config file ( [see sample config file](https://github.com/mitdbg/modeldb/blob/master/client/syncer.json) ) or explicitly via arguments.
+```
+# Initialize syncer from a JSON or YAML config file
```

Comment:
It would be easiest to create the syncer directly in code, so let's give that example and then point to the ModelDB website for other ways of instantiating.

# Create a syncer using a convenience API
syncer_obj = Syncer.create_syncer("Project Name", \
                                  "test_user", \
                                  "project description", \
                                  host="modeldb-backend")